modules/instrument_list/php/instrument_list_controlpanel.class.inc

```diff
@@ -294,7 +294,9 @@ class Instrument_List_ControlPanel extends \TimePoint
                 )
             );
             return;
-        } else {
+        } else if (isset($_REQUEST['setBVLQCStatus'])
```

Comment:
to avoid this kind of confusion in the future could you add one line comment to each of these stating why you added these checks ?

something along the lines of:
`// In order to nullify the value for setBVLQCStatus the front end appends it to the request with an empty string as value`

Reply:
@ridz1208 please resolve if you're happy

Reply:
i changed it again .. its a bit pedantic but now clear